src/internal/m365/collection/groups/collection.go

```diff
@@ -220,31 +217,30 @@ func (col *Collection) streamItems(ctx context.Context, errs *fault.Bus) {
 	semaphoreCh := make(chan struct{}, col.ctrl.Parallelism.ItemFetch)
 	defer close(semaphoreCh)
 
-	// TODO: add for v1 with incrementals
 	// delete all removed items
-	// for id := range col.removed {
-	// 	semaphoreCh <- struct{}{}
+	for id := range col.removed {
+		semaphoreCh <- struct{}{}
 
-	// 	wg.Add(1)
+		wg.Add(1)
 
-	// 	go func(id string) {
-	// 		defer wg.Done()
-	// 		defer func() { <-semaphoreCh }()
+		go func(id string) {
+			defer wg.Done()
+			defer func() { <-semaphoreCh }()
 
-	// 		col.stream <- &Item{
-	// 			id:      id,
-	// 			modTime: time.Now().UTC(), // removed items have no modTime entry.
-	// 			deleted: true,
-	// 		}
+			col.stream <- &Item{
+				id:      id,
+				modTime: time.Now().UTC(), // removed items have no modTime entry.
```

Comment:
can deleted Time be used here? Because it would be the last time item was modified.

Reply:
It could, but we're better off using Now to ensure this item always gets processed.